content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,143 @@
+---
+title: Pody i Węzły
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>Cele</h3>
+                <ul>
+                    <li>Poznać Podamy Kubernetes.</li>
+                    <li>Poznać węzły Kubernetes.</li>
+                    <li>Nauczyć się rozwiązywać problemy z aplikacjami.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h2>Pody Kubernetes</h2>
+                <p>Po stworzeniu Deploymentu w Module <a href="/pl/docs/tutorials/kubernetes-basics/deploy-intro/">2</a>, Kubernetes stworzył <b>Pod</b>, który "przechowuje" instancję Twojej aplikacji. Pod jest obiektem abstrakcyjnym Kubernetes, który reprezentuje grupę jednego bądź wielu kontenerów (takich jak Docker lub rkt) wraz ze wspólnymi zasobami dla tych kontenerów. Zasobami mogą być:</p>
+                <ul>
+                    <li>Współdzielony storage, np. Volumes</li>
+                    <li>Zasoby sieciowe, takie jak unikatowy adres IP klastra</li>
+                    <li>Informacje służące do uruchamiania każdego z kontenerów ⏤ wersja obrazu dla kontenera lub numery portów, które mają być użyte</li>
+                </ul>
+                <p>Pod tworzy model specyficznego dla aplikacji "wirtualnego serwera" i może zawierać różne kontenery aplikacji, które są relatywnie blisko powiązane. Przykładowo, pod może zawierać zarówno kontener z Twoją aplikacją w Node.js, jak i inny kontener dostarczający dane, które mają być opublikowane przez serwer Node.js. Kontenery wewnątrz poda współdzielą adres IP Address i przestrzeń portów, zawsze są uruchamiane wspólnie w tej samej lokalizacji i współdzielą kontekst wykonawczy na tym samym węźle.</p>
```

Comment:
```suggestion
                <p>Pod tworzy model specyficznego dla aplikacji "wirtualnego serwera" i może zawierać różne kontenery aplikacji, które są relatywnie blisko powiązane. Przykładowo, pod może zawierać zarówno kontener z Twoją aplikacją w Node.js, jak i inny kontener dostarczający dane, które mają być opublikowane przez serwer Node.js. Kontenery wewnątrz poda współdzielą adres IP i przestrzeń portów, zawsze są uruchamiane wspólnie w tej samej lokalizacji i współdzielą kontekst wykonawczy na tym samym węźle.</p>

Reply:
How about _adres IP_?

Reply:
`adres IP` is a valid polish translation of the `IP Address`.

In here `adres IP Address` was just incorrect.